hack/sync-api.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage() {
+    cat <<EOF
+Build components:
+    $0 <install-filter-repo|filter-repo|sync-api>
+Build plugins:
+    $0 plugins <filter-repo>
```

Comment:
build plugins?

What is the `$0 install-filter-repo|filter-repo`  for?

Reply:
I think filter-repo is a plugin, it is used in line 79 for push commit to api repo
github action does not contain filter-repo , so we need install it before use

Reply:
What will happen if I execute `./sync-api.sh plugins filter-repo`?

Reply:
i misunderstand the plugins and components, it has deleted